### DISCUSSION TITLE
Lobe Chat 为什么不支持 Ollama 模型的视觉识别？

### DISCUSSION THREAD

**Discussion by AkarinLiu** [lobehub/lobehub#3953] [General Question]
我下载了 `llava-phi3` 模型，但是无法使用这个模型的图片识别功能，不知道是这个功能不支持 Ollama 服务还是模型的问题，如果是，希望官方能够支持。
![image](https://github.com/user-attachments/assets/3ce49e2e-7d57-484a-84db-2262d8e28528)

**Comment by arvinxx** [comment]
你要自己配置下这个模型的能力。https://lobehub.com/zh/docs/self-hosting/advanced/model-list#%E6%89%A9%E5%B1%95%E8%83%BD%E5%8A%9B
